test/StakeManager.js

```diff
@@ -94,6 +94,17 @@ describe('StakeManager', function () {
       assertRevert(tx1, 'AccessControl');
     });
 
+    it('should not be able to stake if stake is less than min stake', async function () {
+      const epoch = await getEpoch();
+      const stakes = ['0', '1', '999'];
```

Comment:
Hey 
Is it necessary to run this loop
as 0,1,999 are always less than 1000
0,1 is not a boundary case

Reply:
Will make changes